### DISCUSSION TITLE
Status on FAPI Implementation

### DISCUSSION THREAD

**Discussion by saltNpepper5** [srsran/srsRAN_Project#87] [General Help]
Hello,

I wanted to know the status of the FAPI implementation for srs5G.  Is it implemented and in use or still in development?  I see in the lib directory there is a fapi and fapi_adapter directories but it wasn't clear to me if it is working or in use as the documentation does not have any information as seen here:

https://docs.srsran.com/projects/project/en/latest/dev_guide/source/interfaces/source/fapi.html.

Thank you!

**Comment by ismagom** [comment]
The MAC and PHY use FAPI interface but avoid packing and unpacking. 

The full FAPI is only used to interface with 3rd party in-line accelerators. At the moment this is part of our commercial offering only.
